src/main/java/stepsdesign/AfterActions.java

```diff
@@ -30,9 +30,13 @@
 public class AfterActions {
 
   @AfterStep
-  public static void tearDown(Scenario scenario) {
-    WebDriver driver = SeleniumDriver.getDriver();
-    byte[] screenshotBytes = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
-    scenario.embed(screenshotBytes, "image/png");
+  public static void takeScreenshot(Scenario scenario) {
```

Comment:
where is takeScreenShot called?

Reply:
It is not called explicitly anywhere. It gets executed after every cucumber test step with`@AfterStep` hook